internal/core/version.go

```diff
@@ -5,7 +5,7 @@ package core
 // For more information please see: https://github.com/mongodb/atlas-sdk-go/blob/main/docs/doc_1_concepts.md
 const (
 	// SDK release tag version.
-	Version = "v20231115010.1.0"
+	Version = "v20231115009.0.0"
```

Comment:
Looks like we have bug with SDK updates.

Reply:
yep, seems it can't go to 11
![goes to 11](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExN2l4YjA1dGhkeHk0Yjk1YW1yZ2UybThpem02YW8yNHk3a3p6MXd3ciZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/LJ8aM1wbV9Gko/giphy-downsized.gif)

Reply:
We can go to 100 now :)